proposals/lip-0071.md

```diff
@@ -281,18 +303,18 @@ This function is used to estimate the expected rewards for a validator assuming
 ##### Execution
 
 ```python
-def getExpectedValidatorRewards(validatorAddress: Address) -> tuple[uint64, uint64, uint64, uint64]:
+def getExpectedValidatorRewards(validatorAddress: Address, height: uint32) -> tuple[uint64, uint64, uint64, uint64]:
     validatorParams = Validators.getValidatorParams()
     totalBftWeight = sum(validator.bftWeight for validator in validatorParams.validators)
 
     let validator be the item of validatorParams.validators with item.address == validatorAddress
     if not validator:
         return (0, 0, 0, 0) 
 
-    if validator.bftWeight > 0:
-        blockReward = minimumRewardActiveValidators + (validator.bftWeight * stakeRewardActiveValidators)//totalBftWeight 
-    else:
-        blockReward = defaultRewardStandByValidators
+    if validator.bftWeight > 0: #active validator
+        blockReward = getMinimumRewardActiveValidators(height) + (validator.bftWeight * getStakeRewardActiveValidators(height))//totalBftWeight
```

Comment:
This reward payout does not guarantee that no more than 101 LSK are paid as rewards in a round. I think we have to strictly cap the inflation, so the reward payout should be changed.
I propose: the first time an active validator forges a block they receive a value proportional to their weight, all other blocks forged in a round give only minimal reward. We should discuss it further.

Reply:
After discussions we decided to not change anything for implementational simplicity.